src/ol/style/IconImage.js

```diff
@@ -188,7 +199,7 @@ class IconImage extends EventTarget {
     if (this.imageState_ == ImageState.IDLE) {
       this.imageState_ = ImageState.LOADING;
       try {
-        /** @type {HTMLImageElement} */ (this.image_).src = this.src_;
+        /** @type {HTMLImageElement} */ (this.image_).srcset = this.src_;
```

Comment:
The srcset attribute is not supported in Internet Explorer 11. Is this change intended if only one src is set?

Reply:
It was a test that => will be removed...